.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+## All Submissions:
+-------------------------------------
+* [ ] Have you followed the guidelines in our Contributing document?
```

Comment:
Can we split this into things that are easy to answer YES/NO to without referring to another MD document? Or maybe add a link to the MD document.